sources/ghostdog/src/main.rs

```diff
@@ -89,6 +96,11 @@ struct MatchNvidiaDriverArgs {
     driver_name: String,
 }
 
+#[derive(FromArgs, PartialEq, Debug)]
+#[argh(subcommand, name = "infiniband-present")]
+/// Detect if Infiniband devices are attached.
+struct InfinibandPresentArgs {}
```

Comment:
nit: Not sure if `argh` likes this, but you could just do

```suggestion
struct InfinibandPresentArgs;
```

Reply:
`argh` doesn't like the change.